src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/organizations.py

```diff
@@ -70,17 +70,11 @@ def update_policy(self, content, policy_id):
             Content=content
         )
 
-    def create_policy(self, content, ou_path, policy_type="SERVICE_CONTROL_POLICY"):
+    def create_policy(self, content, ou_path, policy_name, policy_type="SERVICE_CONTROL_POLICY"):
```

Comment:
The `ou_path` is no longer used in this function. Could you remove that?

Reply:
Done.